caps/pip.md

```diff
@@ -0,0 +1,431 @@
+# Parity Light Protocol (PIP)
+
+The Parity Light Protocol is a variation of LES designed and implemented by Parity Tech for the Parity Ethereum client.
+
+Please refer to the LES specification for information on the purpose of light clients and protocols.
+
+PIP adopts a flow-control mechanism closely analogous to a [token-bucket rate limiter](https://en.wikipedia.org/wiki/Token_bucket) where the client is expected to mirror the server token-bucket state (as exceeding the 'burstiness' depth is a violation that results in disconnection). This is also explained in more detail in the [LES][LES] documentation.
+
+PIP utilises `Canonical Hash Tries` (CHTs), which are also described in the [LES][LES] documentation.

```

Comment:
With 'defined links' like `[LES][LES]` here, it's sufficient to just put `[LES]`. The link will still work. Please adopt this convention. The benefit is that the plain-text representation is less cluttered.